### PR TITLE
feat: import-export type and atom

### LINKED ISSUE #1523: Import Export Type
- [ ] Remove `admin.resolver`

## `AdminService.export`

On the frontend we'll fetch `Array<InterfaceType>`, and another query to get the array of descendant id types. Then store query results as json.

## `AdminService.import`

We'll parse the array of types and create them, starting with the types first before creating the interface.

---

Some notes

- I think generating uuid in frontend seems like the way to go
- There is some old code for import/export that would need to be cleaned up, such as getTypeGraph cypher etc, `graph: TypeGraph!` on the type.schema itself

### PR DESCRIPTION
**Pull request by antonmihaylov**
fixes #1523 #1532

### CODE REVIEW

File: apps/web/src/graphql/repositories/admin/exportAdminData.cypher [hunk 1]
```diff
@@ -11,7 +11,7 @@ MATCH (this)-[r]-() WHERE labels(this)[0] ENDS WITH 'Type' OR labels(this)[0]='E
     vertices:  [node IN nodes | apoc.map.merge(properties(node), {__resolveType: labels(node)[0], typeKind: labels(node)[0]})],
     edges: apoc.coll.union(
         [rel IN relationships WHERE labels(startNode(rel))[0] = 'InterfaceType' |
-                apoc.map.merge(properties(rel), { source: startNode(rel).id, target: endNode(rel).id, __resolveType: 'InterfaceTypeEdge'})],
+                apoc.map.merge(properties(rel), { source: startNode(rel).id, target: endNode(rel).id, __resolveType: 'Field'})],
```
Comment (webberwang) [inline apps/web/src/graphql/repositories/admin/exportAdminData.cypher:14]: Actually thinking of changing this graph structure to using descendent id approach, so we don't need to have a complicated cypher query like this

File: libs/frontend/modules/type/src/store/type-import.service.ts [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+        return foundPrimitive
+      }
+    }
+
+    // if not -get by id
+    return typeService.type(importedType.id)
+  }
+
+  /// Makes the create input without any of the type-type relations
+  private makeCreateInput(
```
Comment (webberwang) [inline libs/frontend/modules/type/src/store/type-import.service.ts:201]: I would assume these can be re-used for normal type create? We could have a shared TypeInputFactory?

**Review by webberwang — CHANGES_REQUESTED** [review]

File: apps/web/src/graphql/repositories/admin/exportAdminData.cypher (hunk 1 above)
Comment (antonmihaylov) [reply apps/web/src/graphql/repositories/admin/exportAdminData.cypher:14]: I didn't use this at all, only regular generated graphql endpoints. I.e. the descendantIds approach

File: libs/frontend/modules/type/src/store/type-import.service.ts (hunk 2 above)
Comment (antonmihaylov) [reply libs/frontend/modules/type/src/store/type-import.service.ts:201]: Good idea!

File: apps/web/src/graphql/repositories/admin/exportAdminData.cypher (hunk 1 above)
Comment (webberwang) [reply apps/web/src/graphql/repositories/admin/exportAdminData.cypher:14]: yeah let's remove it

**Pull request merged**